docs/core/diagnostics/available-counters.md

```diff
@@ -83,7 +83,7 @@ The following counters are published as part of the [ASP.NET Core Kestrel web se
 
 ## System.Net.Http counters
 
-The following counters are published by the HTTP stack.
+The following counters are published by the HTTP stack and are mantained in  [`HttpTelemetry.AnyOS.cs`](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Net.Http/src/System/Net/Http/HttpTelemetry.AnyOS.cs).
```

Comment:
```suggestion
The following counters are published by the HTTP stack and are maintained in [_HttpTelemetry.AnyOS.cs_](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Net.Http/src/System/Net/Http/HttpTelemetry.AnyOS.cs).
```

Reply:
Thanks @IEvangelist should I update all the other links for consistency?

Reply:
Yeah, whenever we refer to source files for example, they should be italicized. And inline code doesn't actually get rendered as such when it's a link.